spec/requests/medications_spec.rb

```diff
@@ -91,39 +96,19 @@ def post_create(medication_params)
             .to change(Medication, :count).by(1)
         end
 
-        context 'when refill exists' do
-          it 'has no validation errors and has correct refill' do
-            post_create valid_medication_params
-            expect(assigns(:medication).errors).to be_empty
-            expect(response.body).to include(valid_medication_params.refill)
-          end
-        end
-
-        context 'when refill does not exist' do
-          it 'has no validation errors and uses auto-generated refill' do
-            post_create valid_medication_params.merge(refill: nil)
-            expect(assigns(:medication).errors).to be_empty
-            expect(assigns(:medication)[:refill]).to be_between(assigns(:medication)[:created_at].to_date,
-              ((7 * assigns(:medication)[:total]) / (assigns(:medication)[:dosage] * assigns(:medication)[:weekly_dosage].count)).days.from_now)
-          end
-        end
-
-        it 'redirects to the medication page' do
+        it 'redirects to medication' do
           post_create valid_medication_params
-          expect(response).to redirect_to medication_path(valid_medication_params.name)
+          expect(response).to have_http_status(302)
         end
       end
 
       context 'when invalid params are supplied' do
-        let(:invalid_medication_params) { valid_medication_params.merge(name: nil, dosage: nil) }
-        before { post_create invalid_medication_params }
-
-        it 're-renders the new medication form' do
-          expect(response).to render_template(:new)
-        end
-
-        it 'adds errors to the medication ivar' do
-          expect(assigns(:medication).errors).not_to be_empty
+        let(:invalid_medication_params) {
+          valid_medication_params.merge(name: nil, dosage: nil)
+        }
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#single-line-blocks-delimiters
```suggestion
        let(:invalid_medication_params) do
          valid_medication_params.merge(name: nil, dosage: nil)
        end
```